app/models/payment/braintree.rb

```diff
@@ -30,9 +30,7 @@ def write_customer(bt_customer, bt_payment_method, member_id, existing_customer)
     end
 
     def customer(email)
-      customer = Payment::Braintree::Customer.find_by(email: email)
-      return customer if customer.present?
-      member = Member.find_by(email: email)
+      member = Member.find_by_email(email)
       member.try(:customer)
```

Comment:
Thanks, @rodrei. I'd like to move away from storing the email address in more than one location. A customer will always have an associated member, so we'll still get back the same customer (if it exists).